meteor/lib/rundownNotifications.ts

```diff
@@ -142,10 +142,19 @@ export function getBasicNotesForSegment(
 		)
 	}
 
-	if (segment.orphaned === 'deleted') {
+	if (segment.orphaned) {
+		let baseMessage: string
+		switch (segment.orphaned) {
+			case SegmentOrphanedReason.DELETED:
+				baseMessage = 'Segment no longer exists in {{nrcs}}'
+				break
+			case SegmentOrphanedReason.HIDDEN:
+				baseMessage = 'Segment was hidden in {{nrcs}}'
+				break
+		}
 		notes.push({
 			type: NoteSeverity.WARNING,
-			message: generateTranslation('Segment no longer exists in {{nrcs}}', {
```

Comment:
This wont generate translations anymore. The i18n parser cant follow the routes, it needs the strings to be fixed as parameters to the function. 
Make baseMessage be `ITranslatableMessage` instead